metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -163,8 +165,9 @@ class MetalsLanguageServer(
   private val recentlyFocusedFiles = new ActiveFiles(time)
   private val languageClient = new DelegatingLanguageClient(NoopLanguageClient)
   var userConfig: UserConfiguration = UserConfiguration()
-  val excludedPackageHandler: ExcludedPackagesHandler =
-    new ExcludedPackagesHandler(userConfig.excludedPackages)
+  var excludedPackageHandler: ExcludedPackagesHandler =
```

Comment:
Why do we need to have it as a var ? Can't we keep the state inside ExcludedPackagesHandler?

Reply:
We can. I did it in the same manner as other things in MetalsLanguageServer